test/Microsoft.ML.Tests/Scenarios/PipelineApi/TrainSaveModelAndPredict.cs

```diff
@@ -38,5 +41,81 @@ public async void TrainSaveModelAndPredict()
             Assert.True(singlePrediction.Sentiment);
 
         }
+
+        [Fact]
+        public async void TensorFlowTransformTrainSaveModelAndPredict()
```

Comment:
uni test to verify save and restore of unfrozen models with Learning Pipeline API